bin/gwd/gwd.ml

```diff
@@ -1232,14 +1232,14 @@ let make_conf from_addr request script_name env =
        if !images_url <> "" then !images_url
        else if !(Wserver.cgi) then
           begin match Sys.getenv_opt "GW_STATIC_PATH" with
-          | Some x -> x ^ "../images"
-          | None -> "../distribution/gw/images/"
+          | Some x -> x ^ String.concat Filename.dir_sep [".."; "images"]
```

Comment:
I think `x` should be in the list to concatenate too

thanks!

Reply:
Yes, I spotted this one. I am reviewing the CGI install stuff!!